lib/web_ui/test/canvaskit/initialization/does_not_mock_module_exports_test.dart

```diff
@@ -18,13 +18,6 @@ void testMain() {
       // Initialize CanvasKit...
       await bootstrapAndRunApp();
 
-      // CanvasKitInit should be defined...
-      expect(
-        js_util.hasProperty(domWindow, 'CanvasKitInit'),
```

Comment:
`CanvasKitInit` is now the `default` canvaskit export, right?

Reply:
Yes, but that isn't hanging off the window or accessible as a global, so we can't test that anymore from this scope.

Reply:
(And it's much better!)